src/cmd/run/run.go

```diff
@@ -70,26 +74,53 @@ func run(ctx context.Context, names []string, options Options) error {
 		logging.Log("using image: %s", yeyContext.Image)
 	}
 
+	if options.Verbose {
+		fmt.Fprintf(os.Stderr, "context:\n--\n%v--\n", yeyContext)
+	}
+
+	// Container name
 	containerName := yey.ContainerName(contexts.Path, yeyContext)
+	if options.Verbose {
+		fmt.Fprintf(os.Stderr, "container: %s\n", containerName)
+	}
 
+	// Reset
 	if options.Reset {
+		if options.Verbose {
+			fmt.Fprintf(os.Stderr, "removing container first")
+		}
 		if err := docker.Remove(ctx, containerName); err != nil {
 			return fmt.Errorf("failed to remove container %q: %w", containerName, err)
 		}
 	}
 
+	var runOptions []docker.RunOption
```

Comment:
we can probably do away with creating an options slice since all options accept their values. we can pass the empty string to WithWorkDir because even if we dont't pass the option it probably initialises the backing option to empty string anyways.